qiita_db/util.py

```diff
@@ -679,6 +681,36 @@ def purge_filepaths(conn_handler=None):
                 remove(fp)
 
 
+def move_filepaths_to_upload_folder(study_id, filepaths, conn_handler=None):
+    r"""Goes over the filepaths list and moves all the filepaths that are not
+    used in any place to the upload folder of the study
```

Comment:
I'm not seeing in the code anything that covers the "...not used in any place..." aspect of the doc string. Is this accurate?